### DISCUSSION TITLE
Add border to specific window

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#2430] [Q&A]
For example, how should I give the border only to the `guake`?

**Comment by vaxerski** [comment]
you can manip border size and color per window with hyprctl setprop and/or windowrules

**Reply by mshzsh** [reply]
I can't find the `bordersize` option in the docs, can you give me an example with windowrules please?

**Reply by sweetbbak** [reply]
https://wiki.hyprland.org/Configuring/Window-Rules/
windowrulev2 = noborder, class:^(swayimg.*)$, title:^(swayimg.*)$,

It's all right there in the docs man
just set a rule for no borders to match windows that you dont want to have borders

**Reply by mshzsh** [reply]
I don't want any window to have borders except for one exception.
For example, if we set `general:border_size=5` and define an `windowrule=noborder...` for all windows, all windows have a margin 5px! (of course, this is not wise! Even if we write a regex for all windows.).
I want to set `general:border_size=0` and set border 5px to only a specific window, like this:  `windowrule=bordersize 5,^(guake)$`.

I have read the documentation completely.

**Reply by vaxerski** [reply]
added a `bordersize` prop to `hyprctl setprop`. This should solve your problem
